### DISCUSSION TITLE
What should we do of the in-context translation system ?

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-server#10642] [Q&A]
What should we do of the in-context translation system ?
- Recommit to it, and do manually (or automatically each week) PRs ?
- Make it use Taxonomy Editor API ?
![image](https://github.com/user-attachments/assets/4d360e3c-34db-417e-b2ea-a9e19dfed6a3)

https://world-de.openfoodfacts.org/labels?translate=add

cc @alexgarel

**Comment by alexgarel** [comment]
We should deprecate it, clearly, because we quite never import entries in it in taxonomies, and it's better to keep product opener more lightweight.

I think if we find a way to create a project each month dedicated to translations, we could try to then send to search in taxonomy editor for untranslated entries (eg. https://ui.taxonomy.openfoodfacts.org/food_ingredients/2024_07_26_food_ingredients_alex_round_trip/search?q=language%3Anot%3Asq+)
